CMakeLists.txt

```diff
@@ -451,6 +451,17 @@ set_target_properties(flint
     RUNTIME_OUTPUT_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/bin"
 )
 
+# Install PkgConfig file
+set(prefix ${CMAKE_INSTALL_PREFIX})
+set(exec_prefix "\$\{prefix\}")
+set(includedir "\$\{prefix\}/include/flint")
```

Comment:
That's supposed to be just `${prefix}/include`, no?

Reply:
Yes, the result in pkgconfig file should be like this. That's Why I put backslash before `$` and `{` so It doesn't replaced by the variable above.

Reply:
I mean, it's not supposed to be suffixed by `/flint`.

Reply:
Take a look at flin.pc.im

Reply:
For reference, this is my `flint.pc`:
```
prefix=/home/albin/cbuilds
exec_prefix=${prefix}
includedir=${prefix}/include
libdir=${exec_prefix}/lib

Name: FLINT
Description: Fast Library for Number Theory
Version: 3.1.0-dev
URL: https://flintlib.org/
Requires: gmp >= 6.2.1 mpfr >= 4.1.0
Cflags: -I${includedir}
Libs: -L${libdir} -lflint
```

Reply:
You mean users are supposed to include flint headers like `#include "flint/flint.h"` instead of `#include "flint.h"`

Reply:
Yes. See https://flintlib.org/doc/building.html#linking-and-running-code

Reply:
fixed.